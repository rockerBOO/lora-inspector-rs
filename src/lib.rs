mod file;
mod metadata;
mod network;
mod norms;
mod weight;
mod worker;

#[derive(Debug)]
pub enum Error {
    Candle(candle_core::Error),
    SafeTensor(safetensors::SafeTensorError),
    Load(String),
    Msg(String)
}
