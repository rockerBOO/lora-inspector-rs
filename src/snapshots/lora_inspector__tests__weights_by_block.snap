---
source: src/lib.rs
expression: result
---
{
  "unet": {
    "up_02": 2.120655256833002,
    "down_01": 2.230538075096385,
    "mid_02": 2.224794629779769,
    "up_01": 2.2393360019167305,
    "up_00": 2.1742004173330853,
    "down_02": 2.2030147237448237,
    "mid_01": 2.2196846091685765,
    "mid_00": 2.1526473038635956,
    "down_00": 2.0547019995405065
  },
  "text_encoder": {
    "layer_09": 1.2418230235341936,
    "layer_01": 1.237123584120902,
    "layer_10": 1.2340809383377243,
    "layer_11": 1.2385596831422123,
    "layer_04": 1.2323092332639434,
    "layer_03": 1.232601893524758,
    "layer_07": 1.229626061262309,
    "layer_02": 1.238819093326412,
    "layer_08": 1.233122912380843,
    "layer_05": 1.227648301288342,
    "layer_00": 1.2405343164401639,
    "layer_06": 1.2311805503592268
  }
}
